frontend-web/src/app/components/ui/Sidebar.tsx

```diff
@@ -0,0 +1,97 @@
+'use client';
+import React, { useRef, useState } from 'react';
+
+interface SidebarProps {
+  content: string | null;
+  closeSidebar: () => void;
+}
+
+const Sidebar: React.FC<SidebarProps> = ({ content, closeSidebar }) => {
+  const [isOpen, setIsOpen] = useState(true);
+  const [height, setHeight] = useState(256);
```

Comment:
valeur fixe a éviter.
Sinon utilise une constante qui explique ce que fait cette constante.
Par exemple :
```suggestion
  const SCREEN_HEIGHT = 256;
  const [height, setHeight] = useState(SCREEN_HEIGHT);
```